test/functional/roles_controller_test.rb

```diff
@@ -59,6 +73,25 @@ class RolesControllerTest < ActionController::TestCase
     assert_not_nil Role.find_by_id(roles(:manager).id)
   end
 
+  test 'should disable filter overriding' do
+    permission1 = FactoryGirl.create(:permission, :domain, :name => 'permission1')
+    role = FactoryGirl.build(:role, :permissions => [])
+    role.add_permissions! [ permission1.name ]
+    org1 = FactoryGirl.create(:organization)
+    org2 = FactoryGirl.create(:organization)
+    role.organizations = [ org1 ]
```

Comment:
Duplicate setup code between all 3 new tests here, move to a context (`with_taxonomies` sounds good) and share it